src/omero_cli_render.py

```diff
@@ -284,10 +287,10 @@ def _configure(self, parser):
                           nargs="+")
         set_cmd.add_argument(
             "channels",
-            help="Rendering definition, local file or OriginalFile:ID")
+            help="Rendering settings, local file or OriginalFile:ID")
```

Comment:
We need to reformulate here. I would go for

``Local file or OriginalFile:ID. The file contains the rendering settings.``

Reply:
Maybe ``...which contains...`` or ``...which specifies...``